components/post_view/post_list.jsx

```diff
@@ -252,51 +241,49 @@ export default class PostList extends React.PureComponent {
             }
 
             if (doScrollToBottom) {
-                this.atBottom = true;
-                postList.scrollTop = postList.scrollHeight;
+                postList.scrollTop = postlistScrollHeight;
                 return;
             }
 
             // New posts added at the top, maintain scroll position
-            if (this.previousScrollHeight !== postList.scrollHeight && posts[0].id === prevPosts[0].id) {
-                postList.scrollTop = this.previousScrollTop + (postList.scrollHeight - this.previousScrollHeight);
+            if (snapshot.previousScrollHeight !== postlistScrollHeight && posts[0].id === prevPosts[0].id) {
+                window.requestAnimationFrame(() => {
```

Comment:
Chrome does not correct it well if we scroll fast to the top and cause few extra calls before correcting properly. 
hence the `requestAnimationFrame`. I would have preferred to avoid but i couldn't figure another way and happens only on chrome so i guess it has something to do with chrome scroll optimisations. I only guess is to check this once in a while and remove when chrome corrects it fine.